unlock-protocol.com/src/components/page/PaywallTags.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+export const PaywallTags = ({ lock }) => {
+  if (lock) {
+    return (
+      <React.Fragment>
+        <script
+          src="https://paywall.unlock-protocol.com/static/paywall.min.js"
```

Comment:
this is fine for now, but let's open up an issue to add the `PAYWALL_URL` env variable to unlock-protocol and use that here, and add a TODO here so we don't forget

Reply:
SG!